TextEditors and Form: Add note for labelMode (#3241)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxForm/1 Configuration/labelMode.md

[note]

Please review the following notes:

> - The following [editors](/api-reference/10%20UI%20Components/dxForm/5%20Item%20Types/SimpleItem/editorType.md '/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorType') do not support *"static"* and *"floating"* modes and will fall back on the *"outside"* mode:

Reviewer: ```suggestion
- The following [editors](/api-reference/10%20UI%20Components/dxForm/5%20Item%20Types/SimpleItem/editorType.md '/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorType') do not support *"static"* and *"floating"* modes and will default to *"outside"* mode:
```
